src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good to see PropTypes being used for type checking. However, the `isAnon` prop is declared as a string in PropTypes but it seems to be used as a boolean in the code. Please correct the type.
```diff
-  isAnon: PropTypes.string
+  isAnon: PropTypes.bool
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->